packages/snap/src/keyring.test.ts

```diff
@@ -43,20 +49,27 @@ jest.mock('../src/utils/ethers', () => ({
     new ethers.Wallet(privateKey, ethers.provider),
 }));
 
-jest.mock('uuid', () => ({
-  v4: () => mockAccountId,
-}));
+jest.mock('uuid', () => {
+  return {
+    v4: () => {
+      accountCreationCount += 1;
+      return accountCreationCount === 1
+        ? mockAccountId
+        : jest.requireActual('uuid').v4();
+    },
+  };
+});
 
 // @ts-expect-error Mocking Snap global object
 global.snap = {
   request: jest.fn(),
   emitEvent: jest.fn(),
 };
 
-const defaultState: KeyringState = {
+const getInitialState = () => ({
```

Comment:
We can just keep the return type as `KeyringState`

Reply:
Addressed here [4ddafb2](https://github.com/MetaMask/snap-account-abstraction-keyring/pull/51/commits/4ddafb2bf0554535c9384615cf8743a993f92fd9)